### DISCUSSION TITLE
토비의 스프링 6 강의 정리

### DISCUSSION THREAD

**Discussion by 1jeongg** [BDD-CLUB/challenge-every-week-posting#580] [1. 개발 관련 글 모음]
토비의 스프링 6 강의를 보고 정리한 내용입니다.. 강의를 너무 잘하셔서 3일만에 다 봤습니다 ㅎㅎ

스프링 다시 공부하는데 재밌네요..!

[오브젝트와 의존관계](https://1jeongg.github.io/toby1/)

[테스트](https://1jeongg.github.io/toby2/)

[템플릿](https://1jeongg.github.io/toby3/)

[예외](https://1jeongg.github.io/toby4/)

[서비스 추상화](https://1jeongg.github.io/toby5/)

**Comment by 02ggang9** [comment]
와웅 토비님 강의 좋은가요 ?.? ㅎㅎ..

저도 살까 고민하고 있는데 후기 알고 싶습니다~~

**Reply by 1jeongg** [reply]
저는 엄청 좋았어요! 빈을 사용하는 이유부터 스프링 동작 방식이나 왜? 이 기술을 쓰는지, 아무 생각 없이 쓰던 템플릿이 어떻게 구성되는지 등을 배울 수 있어서 추천드립니다..!

강의 방식은 이론 설명 절반 + 주문 관련 시스템 개발이었습니다. 강의도 잘하셔서 전혀 지루할 틈이 없었어요! 저는 1.2배속으로 자막 켜고 봤습니다.

대 김영한 선생님의 강의와 비교했을 때 스프링을 단기간에 이해하기에 좋은 것 같아요!
